### PR TITLE
Created Sass pipeline based off less pipeline. All tests passing.

### PR DESCRIPTION
**Pull request by mschmidt712**
resolves issue #2  (edit by @obuckley-kenzan)

### CODE REVIEW

**Comment by obuckley-kenzan** [comment]
ok to test

**Comment by obuckley-kenzan** [comment]
ok to test

**Comment by obuckley-kenzan** [comment]
@paul-barry-kenzan / @csharon / @rkunboxed 
Please review

**Comment by obuckley-kenzan** [comment]
ok to test

**Comment by obuckley-kenzan** [comment]
ok to test

**Comment by obuckley-kenzan** [comment]
Can one of the admins verify this patch?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,104 @@
-# pipeline-compile-sass
-A Gulp pipeline for compiling SASS
+## pipeline-compile-sass
+
+
+## Information
+
+| Package       | Description   | Version|
+| ------------- |:-------------:| -----:|
+| pipeline-compile-sass| This pipeline compiles SCSS files and returns CSS files | 1.0.1 |
```
Comment (obuckley-kenzan) [inline README.md]: I think at this point the initial version should be 0.1.0

File: README.md [hunk 2]
```diff
@@ -1,2 +1,104 @@
...
+
+# Overview
+
+This pipeline assists the compilation of SCSS into CSS, while optionally allowing for autoprefixing and concatination.
+
+
+## Install
+`npm install pipeline-compile-sass.git --save-dev`
+
+**note: https://github.com/kenzanlabs/pipeline-compile-sass/issues/7**
```
Comment (obuckley-kenzan) [inline README.md]: copy / paste?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,104 @@
...
+| ------------- |:-------------:| -----:|
+| pipeline-compile-sass| This pipeline compiles SCSS files and returns CSS files | 1.0.1 |
+
+# Overview
+
+This pipeline assists the compilation of SCSS into CSS, while optionally allowing for autoprefixing and concatination.
+
+
+## Install
+`npm install pipeline-compile-sass.git --save-dev`
```
Comment (obuckley-kenzan) [inline README.md]: should remove `.git` here

File: gulpfile.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+var compilePipeline = require('./src/index.js');
+var testPipeline = require('pipeline-test-node');
+var validatePipeline = require('pipeline-validate-js');
+
+var options = {
+  plugins: {
+    istanbul: {
+      thresholds: {
+        global: {
+          branches: 70
```
Comment (obuckley-kenzan) [inline gulpfile.js]: is threshold required to get the build to pass?  By default the threshold is at 80%, so we should only set this if there's an issue achieving that threshold with comments or an issue speaking to what that might be,

File: package.json [hunk 5]
```diff
@@ -0,0 +1,57 @@
+{
+  "name": "pipeline-compile-sass",
+  "version": "1.0.1",
```
Comment (obuckley-kenzan) [inline package.json]: see comments above re: version

File: package.json [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+    "autoprefixer": "6.4.1",
+    "gulp-sourcemaps": "1.6.0",
+    "lazypipe": "1.0.1",
+    "pipeline-handyman": "0.3.0"
+  },
+  "devDependencies": {
+    "gulp": "3.9.0",
+    "gulp-clean": "0.3.2",
+    "gulp-expect-file": "0.0.7",
+    "gulp-plumber": "^1.1.0",
```
Comment (obuckley-kenzan) [inline package.json]: we should lock this dependency down (no `^`)

File: package.json [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+  "license": "Apache-2.0",
+  "dependencies": {
+    "gulp-concat": "2.6.0",
+    "gulp-if": "2.0.0",
+    "gulp-sass": "2.3.2",
+    "gulp-postcss": "6.2.0",
+    "autoprefixer": "6.4.1",
+    "gulp-sourcemaps": "1.6.0",
+    "lazypipe": "1.0.1",
+    "pipeline-handyman": "0.3.0"
```
Comment (obuckley-kenzan) [inline package.json]: I think there is at least a 1.0.0 release available for handyman.  I would consider a fresh install of this package to grab the latest.

File: package.json [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+  },
+  "devDependencies": {
+    "gulp": "3.9.0",
+    "gulp-clean": "0.3.2",
+    "gulp-expect-file": "0.0.7",
+    "gulp-plumber": "^1.1.0",
+    "pipeline-test-node": "1.0.1",
+    "pipeline-validate-js": "1.0.0-rc3"
```
Comment (obuckley-kenzan) [inline package.json]: I think there is an rc4 available for this.  maybe just do a fresh npm install of this package to grab the latest

**Comment by obuckley-kenzan** [comment]
Thanks for the PR @mschmidt712 !  I left a few feedback items for you.  I will look into the build failure (doesn't seem related to the changes presented here)

**Comment by obuckley-kenzan** [comment]
ok to test

**Comment by obuckley-kenzan** [comment]
Also, I'm ok with starting this of at `1.0.0` actually since the pipeline pattern we've established is pretty solid / agreed upon at this time.

Also, for some reason node-sass keeps failing to install.  Hopefully just a temp. glitch in NPM land.

File: package.json [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+    "Pipeline",
+    "SASS"
+  ],
+  "preferGlobal": false,
+  "private": false,
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/"
+  },
+  "engines": {
+    "node": ">=0.10.3",
```
Comment (csharon) [inline package.json]: I would require at least version 4.0.0 of node.

**Comment by paul-barry-kenzan** [comment]
@mschmidt712 
This PR is actually issue number 2. Please update the description to identify the appropriate issue.

cc: @obuckley-kenzan @csharon

File: README.md [hunk 10]
```diff
@@ -1,2 +1,104 @@
-# pipeline-compile-sass
-A Gulp pipeline for compiling SASS
+## pipeline-compile-sass
```
Comment (paul-barry-kenzan) [inline README.md]: Can we make this an H1?

**Comment by paul-barry-kenzan** [comment]
@mschmidt712 
Nice job!

File: package.json (hunk 7 above)
Comment (mschmidt712) [inline package.json]: For some reason my tests fail when I update this package to 1.0.0

Comment (paul-barry-kenzan) [inline package.json]: @mschmidt712 
Can you provide the test error? I believe the internal architecture changed a bit on **most** keystone pipelines when moving to 1.0

**Comment by obuckley-kenzan** [comment]
thanks @paul-barry-kenzan .  @mschmidt712 the contributing guidelines require a bit more formality in the PR process.  please review

**Comment by obuckley-kenzan** [comment]
ok to test

File: package.json (hunk 7 above)
Comment (paul-barry-kenzan) [inline package.json]: @obuckley-kenzan 
Handy man wasn't updated before the merge. Was that intended?

**Comment by obuckley-kenzan** [comment]
@paul-barry-kenzan 
I think it's ok, but agree it would have been nice to have.

**Pull request merged**